docs/config_file.md

```diff
@@ -66,6 +66,7 @@ Chrome, Chrome Canary, Chromium, Firefox, IE, Opera, PhantomJS, Safari, Safari T
 
 ### Config-level options:
 
+    additionalBrowserSocketEvents[Object]  name of events (keys) to an event handling function
```

Comment:
suggestion:
`eventNames (key) to custom event handlers functions to be added to the browser socket`